### ISSUE TITLE
mobile touch

### ISSUE THREAD

**Issue by gnud** [ONE-LOGIC/ngFlowchart#19]
I'm trying to move the nodes via android device, nothing happens.

**Comment by PerFuchs** [comment]
This is not supported and as far as I know it is also not planned in near future.

**Comment by gnud** [comment]
Yes, I can see that, however any ideas on how could someone make a patch for the events?
